docs/tutorials/platform_beginner/7_embed_content.md

```diff
@@ -116,27 +116,25 @@ namespace App\Controller;
 
 use eZ\Bundle\EzPublishCoreBundle\Controller;
 use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+use eZ\Publish\API\Repository\ContentService;
 
 class RideController extends Controller
 {
-    /**
-     * Action used to display a ride
-     *    - Adds the list of all related Landmarks to the response.
-     *
-     * @param ContentView $view
-     *
-     * @return ContentView $view
-     */
+    private $contentService;
+
+    public function __construct(ContentService $contentService)
```

Comment:
Not sure it this is mentioned anywhere, but that will only works with `autowiring` enabled

Reply:
The assumption is that the tutorial is started from a clean install, and it is enabled by default.